### ISSUE TITLE
[BUG/ISSUE] ObsPack diagnostic menu error

### ISSUE THREAD

**Issue by JinZheeee** [geoschem/geos-chem#1364]
Hi,
The model version I used is 12.9.3, and I'm running the CO2 simulation with ObsPack diagnostic.

The diagnostic results are ok except for con_aircraft-flask_42_allvalid and con_aircraft-insitu_42_allvalid simulations. The model simulations for some observations at these two sites are 0.0 ppm, which are not reasonable. I try to calculate the corresponding model simulations by myself, the values are around 400 ppm, not 0 ppm.

Here is part of the record of wrong simulations, including the observation date, rank in the daily file, obspack observation (ppm), and obspack id.
```console
20140908	4373	397.052	co2_con_aircraft-flask_42_allvalid~11444074
20140924	4970	396.751	co2_con_aircraft-flask_42_allvalid~11444098
20150907	5482	399.433	co2_con_aircraft-flask_42_allvalid~11444469
20150927	6228	399.984	co2_con_aircraft-flask_42_allvalid~11444493
20160127	5574	401.215	co2_con_aircraft-flask_42_allvalid~11444599
20160315	5120	401.225	co2_con_aircraft-flask_42_allvalid~11444635
20170727	6938	405.338	co2_con_aircraft-flask_42_allvalid~11444947 
20140909	4196	395.482	co2_con_aircraft-insitu_42_allvalid~25329883
20140912	4548	394.478	co2_con_aircraft-insitu_42_allvalid~25332432
20140914	4972	395.646	co2_con_aircraft-insitu_42_allvalid~25335339
20140917	5555	396.718	co2_con_aircraft-insitu_42_allvalid~25339610
20140918	5000	397.266	co2_con_aircraft-insitu_42_allvalid~25341320
20140918	5001	394.994	co2_con_aircraft-insitu_42_allvalid~25341321
20140919	4960	396.57	co2_con_aircraft-insitu_42_allvalid~25342923
20140921	5145	396.091	co2_con_aircraft-insitu_42_allvalid~25345852
20140922	4959	396.746	co2_con_aircraft-insitu_42_allvalid~25347468
20140925	5329	397.465	co2_con_aircraft-insitu_42_allvalid~25351671
20140926	5475	395.702	co2_con_aircraft-insitu_42_allvalid~25353490
20140926	5476	396.586	co2_con_aircraft-insitu_42_allvalid~25353491
20140929	4688	394.634	co2_con_aircraft-insitu_42_allvalid~25357355
20141003	4518	395.31	co2_con_aircraft-insitu_42_allvalid~25361514
20141003	4519	395.31	co2_con_aircraft-insitu_42_allvalid~25361515
20141003	4520	395.3	co2_con_aircraft-insitu_42_allvalid~25361516
......
````
Can you check and fix this problem?

Look forward to hearing from you.

Thanks.

**Comment by yantosca** [comment]
Thanks for writing @JinZheeee.  Due to travel, I will be unable to look at this for the next couple of weeks, so I am tagging @Jourdan-He and @msulprizio.

We would also recommend trying some of the steps on our [Debug GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/debug-guide.html).

**Comment by yantosca** [comment]
Thanks for your patience @JinZheeee.  I was away for 2 weeks.

There have been a couple of ObsPack bug fixes since version 12.9.3.  See issues  #747 and #744.
